KarhooUISDK/Routing/BaseViewController.swift

```diff
@@ -130,6 +130,10 @@ public extension BaseViewController {
             loadingView.set(backgroundColor: .black, alpha: loadingViewAlpha)
         }
 
-        show ? loadingView.show() : loadingView.hide()
```

Comment:
This was clear enough and had brevity as well. Any particular reason for the change?

Reply:
There was a warning. This syntax is not allowed for Void methods.